apps/music/js/ui/title_bar.js

```diff
@@ -97,25 +97,28 @@ var TitleBar = {
               PlayerView.currentIndex
             ];
             var playingBlob = PlayerView.playingBlob;
-            AlbumArtCache.getCoverBlob(currentFileinfo).then(function(picture) {
-              var currentMetadata = currentFileinfo.metadata;
-              App.pendingPick.postResult({
-                type: playingBlob.type,
-                blob: playingBlob,
-                name: currentMetadata.title || '',
-                // We only pass some metadata attributes so we don't share
-                // personal details like # of times played and ratings.
-                metadata: {
-                  title: currentMetadata.title,
-                  artist: currentMetadata.artist,
-                  album: currentMetadata.album,
-                  picture: picture
-                }
-              });
 
-              cleanupPick();
+            LazyLoader.load('js/metadata/album_art_cache.js', function() {
+              AlbumArtCache.getCoverBlob(currentFileinfo)
+                           .then(function(picture) {
+                var currentMetadata = currentFileinfo.metadata;
+                App.pendingPick.postResult({
+                  type: playingBlob.type,
+                  blob: playingBlob,
+                  name: currentMetadata.title || '',
+                  // We only pass some metadata attributes so we don't share
+                  // personal details like # of times played and ratings.
+                  metadata: {
+                    title: currentMetadata.title,
+                    artist: currentMetadata.artist,
+                    album: currentMetadata.album,
+                    picture: picture
+                  }
+                });
+
+                cleanupPick();
+              });
```

Comment:
I am not sure if there is a good timing to load `js/metadata/album_art_cache.js` so that we don't have to lazy load it every time when we need the `AlbumArtCache` object, or maybe we can use some technique to simplify the `LazyLoader.load` pattern, (eg. Promise) like a async object helper.

It's just a idea and we don't have to do it in this bug, let's discuss more during the music refactoring works.